index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Owesome Books App</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="Container-fluid">
+        <header>
+            <div class="searcg-bar">
+                <h1 class="bar-head">A webPage</h1>
+                <div class="webdiv">
+                    <div class="webicon">  
+                        <ul class="list-Con">
+                            <li class="nav-item" id="Lists">Home</li>
+                            <li class="nav-item" id="Adds" >Back</li>
+                            <li class="nav-item" id="Links">About Us</li>
+                        </ul>
+                    
+                    </div>
+                    <div class="search">
+                    <input type="search" placeholder="https://" class="search input">
+                    <button class="button">search</button>
+                    </div>   
+                </div>
+            </div>
+            <nav class="nav-bar">
+            <section class="logo"><a>Owesome Books</a></section>
+           <ul class="list-Container">
+                <li class="nav-items" id="List">List</li>
+                <li class="nav-items" id="Add" >Add New</li>
+                <li class="nav-items" id="Link">Contact</li>
+            </ul>
+            </nav>
+        </header>
+        <div class="datetime">
+            <p class="date"></p>
```

Comment:
- [ ] Also, it would be better if our date is displayed under the navigation as shown in the template :+1: 

Original
![Screenshot from 2022-09-19 22-57-41](https://user-images.githubusercontent.com/46388113/191133870-7c6ee917-9c36-49b1-9f10-3257c67d69c9.png)

Our Project
![Screenshot from 2022-09-19 22-58-26](https://user-images.githubusercontent.com/46388113/191133912-8c3399de-3b97-4dbf-98ba-f58e3414feb2.png)
 Also try and match the design